package.json

```diff
@@ -118,10 +118,9 @@
     "@testing-library/react": "^11.2.3",
     "@testing-library/react-hooks": "^3.7.0",
     "@testing-library/user-event": "^12.6.0",
-    "@web-stories-wp/e2e-test-utils": "*",
+    "@web-stories-wp/e2e-tests": "*",
     "@wordpress/babel-plugin-import-jsx-pragma": "^2.7.0",
     "@wordpress/dependency-extraction-webpack-plugin": "^2.9.0",
-    "@wordpress/e2e-test-utils": "^4.16.1",
     "@wordpress/eslint-plugin": "^7.4.0",
     "@wordpress/jest-console": "^4.0.1",
     "@wordpress/jest-puppeteer-axe": "^3.0.1",
```

Comment:
These two can be moved to `e2e-tests/package.json`

Reply:
Same for `@percy/puppeteer`

Reply:
Are these not used by the karma tests?

Reply:
Do you mean remove these from this file, as they are already in new package.json file, [here](https://github.com/google/web-stories-wp/pull/6305/files#diff-73662dce866d908da409d58c0dd1b3fcef7016ad2125dbb7b01a4172249f5280R12)
[here](https://github.com/google/web-stories-wp/pull/6305/files#diff-73662dce866d908da409d58c0dd1b3fcef7016ad2125dbb7b01a4172249f5280R15)

Reply:
Yes, removed from this file.

And no, they're not used by the Karma tests.

Reply:
Thanks for being clear. Removed in https://github.com/google/web-stories-wp/pull/6305/commits/f5f355d1f300c087b7b43bccf3deccd28de5ed96